src/vendors/eth2.0-deposit-cli-1.2.0/README.md

```diff
@@ -106,34 +106,34 @@ or run the following command to enter the interactive CLI and generate keys from
 
 The CLI offers different commands depending on what you want to do with the tool.
 
-| Command | Description |
-| ------- | ----------- |
-| `new-mnemonic` | (Recommended) This command is used to generate keystores with a new mnemonic. |
+| Command             | Description                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Can you remove all of these changes?  Maybe they happened automatically in your editor?  Since this code is copied as-is from the deposit cli repo, I'd like to leave it the same.

Reply:
Fixed! Apologies - saw markdown table and jumped on formatting it.